README.md

```diff
@@ -19,24 +19,34 @@ Opni currently features log anomaly detection for Kubernetes.
   * Ability to setup & send alerts (slack/email/etc) based on Opni log insights
 
 ----
```

Comment:
What about putting a heading over this entire section titled "Getting Started with Opni" with a short explanation "here are two ways to start, 1. Add Opni to your existing cluster, or 2. Try Opni with our pre-built cluster and demo. Then follow with the detailed steps for each subheading using what you have?

Reply:
addressed in https://github.com/rancher/opni/pull/86